articles/iot-edge/how-to-configure-api-proxy-module.md

```diff
@@ -164,6 +164,26 @@ Configure the following module on any **lower layer** for this scenario:
     }
     ```  
 
+## Expose port
+Port 8000 is exposed by default from docker image, if a different port is used make sure you add the ExposedPorts in the deployment
+
+    ```
+    {
+        "ExposedPorts": {
+            "8001/tcp": {}
+        },
+        "HostConfig": {
+            "PortBindings": {
+                "8001/tcp": [
+                    {
+                        "HostPort": "8001"
+                    }
+                ]
+            }
+        }
+    }
+    ```
```

Comment:
This code block shouldn't be indented because it doesn't follow a list item. It won't render correctly. Remove the indentation for the entire code block.